contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -33,6 +33,19 @@ export function loadContentNode(context, id) {
     });
 }
 
+export function loadContentNodeByNodeId(context, nodeId) {
+  const channelId = context.rootState.currentChannel.currentChannelId;
+  return loadContentNodes(context, { '[node_id+channel_id]__in': [nodeId, channelId] })
```

Comment:
This would have to be `[[nodeId, channelId]]` as it's an array of pairs (`nodeId`, `channelId`) represented as an array.

Reply:
Actually - given the way it gets serialized in the query params to the backend, this probably doesn't matter, but it's more consistent if it's a nested array...